braidpool_spec.md

```diff
@@ -260,13 +406,39 @@ reject it for the following reasons:
    in the absence of attackers, cohorts exceeding the k-width happen naturally
    and cannot be prevented.
 
+[kp]
+
+If we want to do a literature review, we should include other proposals
+too.
+
+[/kp]
+
 ## Simple Sum of Descendant Work
 
+[kp]
+
+I still don't see why we still need it. We needed this when in your original
+proposal the beads were not candidate bitcoin blocks. If all beads on braidpool
+are candidate bitcoin blocks, then we don't need the SSDW algorithm.
+
+[/kp]
+
 Within Bitcoin, the "Longest Chain Rule" determines which tip has the most work
 among several possible tips. The "Longest Chain Rule" only works at constant
 difficulty and the actual rule is a "Highest Work" rule when you consider
 difficulty changes.
 
+[kp]
+
+Why doesn't it work the miner chosen difficulty? If we know the approximate
+amount of work that each share represents, then we simply distribute block
+reward by weighted amount of work done.
+
+The epoch is defined as all the shares between two bitcoin blocks found by
+braidpool. There is no loss of weight over time.
+
+[/kp]
+
```

Comment:
With two chains in a fork, one having half the difficulty per block of the other chain, the one with half-difficulty blocks can be "longer" but have less work.